src/components/buttons/submit.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button, Modal, Box, Typography } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { CheckCircle } from '@mui/icons-material';
+import { useWindowSize } from 'usehooks-ts';
+import { useState } from 'react';
+
+export function Submit() {
+  const { width, height } = useWindowSize();
+  const [open, setOpen] = useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  const SubmitButton = styled(Button)({
```

Comment:
Do you think it might make sense to move style out of .tsx files and instead import as modules ?